xilem/src/view/grid.rs

```diff
@@ -195,16 +192,19 @@ impl<W: Widget + FromDynWidget + ?Sized> SuperElement<Pod<W>, ViewCtx> for GridE
         // There is not much else, beyond purposefully failing, that can be done here,
         // because there isn't enough information to determine an appropriate spot
         // for the widget.
-        Self::Child(child.erased(), GridParams::new(1, 1, 1, 1))
+        Self {
+            child: child.erased(),
+            // TODO - Should be 0, 0?
```

Comment:
Ideally we'd follow the rules in [grid-auto-flow](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-auto-flow). But that's again not really for this PR